### PR TITLE
implement ceil function

### PR DESCRIPTION
**Pull request by jaeyoung0909**
Resolves https://github.com/gluesql/gluesql/issues/642

### CODE REVIEW

File: core/src/ast_builder/expr/function.rs [hunk 1]
```diff
@@ -197,8 +208,8 @@ pub fn reverse<T: Into<ExprNode>>(expr: T) -> ExprNode {
 #[cfg(test)]
 mod tests {
     use crate::ast_builder::{
-        abs, acos, asin, atan, col, cos, expr, floor, ifnull, left, num, pi, reverse, right, sin,
```
Comment (ever0de) [inline core/src/ast_builder/expr/function.rs:200]: `reverse` is gone. You need to add this.

File: core/src/ast_builder/mod.rs [hunk 2]
```diff
@@ -23,7 +23,7 @@ pub use expr::{col, expr, nested, num, text, ExprNode};
 pub use expr::{
     aggregate::{max, sum, AggregateNode},
     function::{
-        abs, acos, asin, atan, cos, floor, ifnull, left, pi, reverse, right, sin, tan, upper,
```
Comment (ever0de) [inline core/src/ast_builder/mod.rs:26]: Same here

Comment (jaeyoung0909) [reply core/src/ast_builder/mod.rs:26]: I added `reverse`!

**Review by ever0de — APPROVED** [review]
Looks good 👍

**Review by MRGRAVITY817 — APPROVED** [review]
LGTM 👍

**Pull request merged**